seqr/utils/elasticsearch/es_search.py

```diff
@@ -824,6 +825,16 @@ def _parse_genome_versions(self, result, index_name, hit):
             'liftedOverPos': lifted_over_pos,
         })
 
+    def _parse_transloc_xstop(self, result, index_name):
```

Comment:
the name of this function is a bit confusing, I think just `_parse_xstop` is fine